frontend/lib/parsing/chpl.lex

```diff
@@ -240,6 +240,7 @@ zip              return processToken(yyscanner, TZIP);
 "reduce="        return processToken(yyscanner, TASSIGNREDUCE);
 
 "init="          return processToken(yyscanner, TINITEQUALS);
+"init"           return processToken(yyscanner, TINIT);
```

Comment:
Not a biggy, and probably not worth making the diff bigger over (and you're obviously following precedent here), but it seems too bad that `init`, `init=`, and `reduce=` are orphaned down here rather than being up in the otherwise alphabetized token list.  I imagine that the rationale was that these are "operator-like", but I'm not sure I buy that...